anta/cli/utils.py

```diff
@@ -33,8 +33,9 @@ def setup_logging(level: str = "info") -> None:
 
     logging.getLogger("anta.inventory").setLevel(loglevel)
     logging.getLogger("anta.result_manager").setLevel(loglevel)
+    logging.getLogger("anta.reporter").setLevel(loglevel)
+    logging.getLogger("anta.runner").setLevel(loglevel)
```

Comment:
I forgot to add the logging settings of the new test cases.

Ex:

`logging.getLogger("anta.tests.aaa").setLevel(logging.ERROR)`

Can you add them in this PR please? Thanks :)